mnc/xengine_beamformer_control.py

```diff
@@ -21,6 +21,9 @@
 from astropy.constants import c as speedOfLight
 speedOfLight = speedOfLight.to('m/ns').value
 
+print (speedOfLight)
```

Comment:
Looks like a debugging statement. Could we remove it?

Reply:
Done